Source/WebKit/CMakeLists.txt

```diff
@@ -687,6 +687,12 @@ set(WebCore_SERIALIZATION_IN_FILES
     PlatformEvent.serialization.in
     PlatformMediaSession.serialization.in
     PlatformScreen.serialization.in
+    PlatformWheelEvent.serialization.in
+    PlaybackSessionModel.serialization.in
+    ProtectionSpaceBase.serialization.in
+    ScrollTypes.serialization.in
+    SoupNetworkProxySettings.serialization.in
```

Comment:
Port specific stuff like this shouldn't be in the main `CMakeLists.txt`. In this case it should be in the `PlatformGTK.cmake` and `PlatformWPE.cmake` files.